.github/workflows/release.yml

```diff
@@ -316,13 +316,11 @@ jobs:
         run: yarn global add vuepress && yarn install && yarn docs:build && touch docs/.vuepress/dist/.nojekyll
 
       - name: Deploy 🚀
-        uses: JamesIves/github-pages-deploy-action@releases/v3
+        uses: JamesIves/github-pages-deploy-action@releases@v4.1.5
         with:
           ACCESS_TOKEN: ${{ secrets.gh_access_token }}
           BRANCH: gh-pages # The branch the action should deploy to.
           FOLDER: docs/.vuepress/dist # The folder the action should deploy.
           TARGET_FOLDER: ${{ env.DOCS_VERSION_PATH }} # The folder the action should deploy to.
-          GIT_CONFIG_EMAIL: guillaume.alleon@gmail.com
           COMMIT_MESSAGE: publish documentation
           CLEAN: true
-          SINGLE_COMMIT: true
```

Comment:
You must explain in the commit message why SINGLE_COMMIT is dropped.  This is certainly not a maintenance action.

Reply:
Okay, now let us talk about the CLEAN keyword.  At the moment, master and releases will delete each other, which is not very nice.
We could set CLEAN to false in both workflows, but this sounds suboptimal.  I believe that this should work:
 * set CLEAN to true in build.yml with CLEAN_EXCLUDE: ["version/*"]
 * set CLEAN to false in release.yml, there is normally nothing to clean there

Reply:
done - `CLEAN` is `true` by default